presentation/src/main/java/me/androidbox/presentation/login/screen/LoginScreen.kt

```diff
@@ -0,0 +1,148 @@
+package me.androidbox.presentation.login.screen
+
+import android.util.Log
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.text.ClickableText
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.*
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import me.androidbox.component.R
+import me.androidbox.component.general.PasswordEntry
+import me.androidbox.component.general.TaskButton
+import me.androidbox.component.general.UserInputEntry
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.backgroundInputEntry
+import me.androidbox.presentation.login.viewmodel.LoginViewModel
+
+@Composable
+fun LoginScreen(
+    modifier: Modifier = Modifier,
+    loginViewModel: LoginViewModel = hiltViewModel()
+) {
+
+    Column(modifier = modifier
+        .fillMaxSize()
+        .background(Color.Black)) {
+
+        Spacer(modifier = modifier.height(48.dp))
```

Comment:
You are reusing the main modifier on all composables. If you receive a modifier from parameter, you should use it on the Main Composable (In this case, the Column at line 36)